src/components/Pages/Logout.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Logout() {
```

Comment:
- [ ]  Please, kindly refactor this functional component ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Logout= () => {
  // ...
}
```

You can still do the export at the end of the file.

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪